src/Components/FruitBlock.js

```diff
@@ -0,0 +1,49 @@
+import Product
+    from "./Product";
+import React
+    , {
+    useState
+} from "react";
+
+function FruitBlock(){
+    //global state voor alle counters als beginpunt (+ voor reset)
+    const [initialCount, setInitialCount] = useState(0);
+
+
+    const forceUpdate = React.useCallback(() => setInitialCount(0), []);
+
+    function resetCounters() {
+        setInitialCount(1);
+        setInitialCount(0);
```

Comment:
In principe zou
```suggestion
        setInitialCount(0);
```
voldoende moeten zijn